doc/contributing/sharing-project-news.md

```diff
@@ -0,0 +1,30 @@
+# Sharing project news
+
+The project aims to make it easy
+to share project news with those who publish newsletters
+that cover Node.js.
+
+The project's approach is as follows:
+
+* The project will create one "What's new in YYYY" discussion item in
+  the `nodejs/node` repo for news related to Node.js core.
```

Comment:
I think it might be better to use a separate repo for these news. In this way, publications could subscribe to the notifications on that repository to receive the updates.

Reply:
ok!!